components/scream/src/share/tests/common_physics_functions_tests.cpp

```diff
@@ -98,6 +98,7 @@ void run(std::mt19937_64& engine)
   view_1d temperature("temperature",num_mid_packs),
           height("height",num_mid_packs),
           qv("qv",num_mid_packs),
+          rand_int("rand_int",num_mid_packs),
```

Comment:
This view seems unused.

Reply:
You are right (thanks!), I removed it in a new commit. It looks like, when packs are used, all numbers (real or integers) in a pack are same. Is that a known issue?